### PR TITLE
Update standards position on Trusted Types - fixes #20

### LINKED ISSUE #20: Trusted Types
* Specification Title: Trusted Types for DOM Manipulation
* Specification or proposal URL: https://github.com/WICG/trusted-types/blob/master/README.md

It seems useful to figure out something here early given its impact across a wide range of standards.

### PR DESCRIPTION
**Pull request by mozfreddyb**

### CODE REVIEW

File: activities.json [hunk 1]
```diff
@@ -1573,8 +1573,8 @@
...
     "id": "trusted-types",
     "mozBugUrl": null,
-    "mozPosition": "neutral",
-    "mozPositionDetail": "The API could be used to harden sites against certain cross-site scripting issues, but it is sufficiently complex that we are concerned that it will not be suitable for many sites.",
+    "mozPosition": "positive",
+    "mozPositionDetail": "Mozilla believes that preventing DOM-based XSS is an important security goal. The track record of preventing DOM-based XSS is convincing. That being said, the Trusted Types specification is also providing API methods with little or unknown value and uptake, like getPropertyType, getAttributeType. Additionally, there are features in the Chrome implementations that are not yet standardized, like the beforepolicycreation event. These should ideally be properly standardized (based on a proven need) or deprecated and removed. Dealing with inscrutable third-party dependencies or external JavaScript has been a major concern of security and enforcing reasonable boundaries is a promising approach. We believe that runtime checking of untrusted HTML might pair well with a Sanitizer API that should be complementary.",
```
Comment (martinthomson) [inline activities.json:1577]: The track record of *what* preventing DOM-based XSS?

This seems a little overwrought for a positive position.  It reads as somewhat negative even.

Can you break down the critique on the issue and provide a shorter summary here?

Comment (mozfreddyb) [reply activities.json:1577]: I shortened it. How about this?

> Mozilla believes that preventing DOM-based XSS is an important goal and the track record of Trusted Types preventing it is convincing. The specification will likely require some work towards a clearer API shape. Also, Chrome seems to have added feature that are not yet specified, like the beforepolicycreation event. These should ideally be properly standardized (based on a proven need) or deprecated and removed. We believe that runtime checking of untrusted HTML might pair well with a Sanitizer API that should be complementary.

Comment (martinthomson) [reply activities.json:1577]: Much better.  How about a little shorter yet:

```suggestion
    "mozPositionDetail": "Mozilla believes that preventing DOM-based XSS is an important security goal. The track record of preventing DOM-based XSS is convincing. Dealing with inscrutable third-party dependencies or external JavaScript has been a major concern of security and enforcing reasonable boundaries is a promising approach. We have some reservations about some features in the Chromium implementation, which need to be validated and standardized or removed.",
```

I struggled with the sentence about Sanitizer, but I don't think that we need to talk about Sanitizer here.  That's a comment that might go well on #20 though.

The same applies to:

> That being said, the Trusted Types specification is also providing API methods with little or unknown value and uptake, like getPropertyType, getAttributeType. Additionally, there are features in the Chrome implementations that are not yet standardized, like the beforepolicycreation event. These should be properly standardized (based on a proven need) or deprecated and removed.

Those details can go on the issue discussion (#20).

Comment (mozfreddyb) [reply activities.json]: Agreed. I'll respond there!

**Review by martinthomson — APPROVED** [review]

**Pull request merged**